src/detect-rfb-secresult.c

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2020 Open Information Security Foundation
+/* Copyright (C) 2021 Open Information Security Foundation
```

Comment:
Thanks for taking the time to also update the copyright year :)

One thing is that they must be updated in such a way that it covers their existence, if I can put it that way. That means that here we should have `2020-2021`, not erasing `2020`.